api/sql/schema/750$transfer.push.create.sql

```diff
@@ -105,6 +105,59 @@ BEGIN TRY
     WHERE
         partnerId = @ledgerId
 
+    
+    DECLARE @result TABLE(
```

Comment:
Can we
`DECLARE @result transfer.transferTT` 
and 
`OUTPUT INSERTED.* INTO @result`
and then
`SELECT *,
        @merchantMode,
        REPLACE(REPLACE(REPLACE(CONVERT(varchar, @merchantSettlementDate, 120),'-',''),':',''),' ',''),
        @merchantSerialNumber,
        @merchantSettings,
        @issuerMode,
        REPLACE(REPLACE(REPLACE(CONVERT(varchar, @issuerSettlementDate, 120),'-',''),':',''),' ',''),
        @issuerSerialNumber,
        @issuerSettings,
        @issuerPort,
        @ledgerPort,
        @ledgerMode,
        @ledgerSerialNumber
FROM @result
`